models/articles.models.js

```diff
@@ -59,8 +59,29 @@ function selectCommentsFromArticle(articleID) {
     });
 }
 
+function insertCommentToArticle(articleID, requestBody) {
+  if (!/^[0-9]+$/.test(articleID)) {
+    return Promise.reject({ status: 400, msg: "bad request" });
```

Comment:
You can remove the regex check because the database will throw an error if an invalid article ID is passed. Instead, use Promise.reject() for example when checking for missing or invalid data in the request body (e.g., missing username or body).